File: armcoordination.js
// GitHub: EntityPlantt/Kattis
var x, y, r;
require("readline").createInterface({input: process.stdin}).on("line", line => {
	line = line.split(" ").map(e => parseInt(e));
	if (x == undefined) {
		x = line[0];
		y = line[1];
	}
	else {
		r = line[0];
		console.log(x - r, y - r);
		console.log(x + r, y - r);
		console.log(x + r, y + r);
		console.log(x - r, y + r);
		process.exit();
	}
});

File: armcoordination.cpp
// GitHub: EntityPlantt/Kattis
#include <iostream>
using namespace std;
int main() {
    int x, y, r;
    cin >> x >> y >> r;
    cout << x - r << ' ' << y - r << '\n' << x + r << ' ' << y - r << '\n' << x + r << ' ' << y + r << '\n' << x - r << ' ' << y + r;
    return 0;
}